form/unit_testing.rst

```diff
@@ -46,17 +46,21 @@ The simplest ``TypeTestCase`` implementation looks like the following::
                 'test' => 'test',
                 'test2' => 'test2',
             );
-
-            $form = $this->factory->create(TestedType::class);
-
+           
+           $objectToCompare = new TestObject();
+           //$objectToCompare will retrieve data from the form submission pass it at second argument
```

Comment:
missing space after `//`

Reply:
and 1 space before `//`, 1 space before `$objectToCompare` (the indents are incorrect).